pmm-app/tests/pages/pmmInventoryPage.js

```diff
@@ -62,15 +62,15 @@ module.exports = {
     const nodeId = await this.getNodeId(serviceName);
     I.click(agentLinkLocator);
     // eslint-disable-next-line max-len
-    let flagExists = `//tr[td[contains(text(), "${nodeId}")]]//span[contains(text(),"enhanced_metrics_disabled: true")]`;
+    let flagExists = `//tr//td//span[contains(text(), "${nodeId}")]/../span[contains(text(),"enhanced_metrics_disabled: true")]`;
```

Comment:
I would not reuse the same variable here, especially because those two mean different things:
```suggestion
    const enhanceMetricsDisabled = `//tr//td//span[contains(text(), "${nodeId}")]/../span[contains(text(),"enhanced_metrics_disabled: true")]`;
```